spec/features/availability_spec.rb

```diff
@@ -36,7 +36,7 @@
       visit "/catalog/8606339"
       sleep 5.seconds
       expect(page.all('.location--online .umlaut .fulltext').length).to eq 1
-      expect(page.all('.location--online .umlaut .fulltext .response_item').length).to eq 5
+      expect(page.all('.location--online .umlaut .fulltext .response_item').length).to be >= 4
```

Comment:
That is my fault. I have a slightly different configuration for the app on getit-dev and getit for a feature collapses duplicate listings from the same vendor. Getit-dev is more generous in this match. Sorry for the confusion.